docs/osf-trivia-list/dasharo.md

```diff
@@ -0,0 +1,220 @@
+product called [Dasharo Tools Suite OS][dtsos], which will largely improve firmware
+deployment, hardware compatibility reporting and  binary blobs transmission
+wherever it is necessary, as well as recovery.
+* Firmware update - we are registered [consultants for fwupd/LVFS][lvfs] and enable
+customers and community platforms, so they can get seamless firmware update in
+Linux.
+* Support  - Dasharo team is paid to provide support in community through
+[Dasharo Matrix Space][dasharo-matrix], of course we want to introduce some
+commercial products related to support to make business feasible, but some
+level of support always would be available to community.
+* We provide ready to use binaries with GPG based signing scheme that improve
+verification where firmware coming from.
+
+## What are the differences between the official coreboot repository and the Dasharo repository ?
+
+The difference is support for [compatible
+hardware](https://docs.dasharo.com/#compatible-hardware). Dasharo repository
+contain release tags which are associated with Dasharo Certification Program
+providing certain quality criteria including test results. We always trying to
+minimize delta, but sometimes it can be up to 5k SLOC (or more I guess e.g.
+Talos II coreboot support).
+
+<!--
+
+## Why Dasharo Team is against moving code to branches in coreboot?
+
+## What is the difference between open-source firmware development in free time
+and open-source firmware development as a business?
+
+## What is Dasharo Certification Program?
+-->
+
+## Why there is no AMD mainboard supported in Dasharo ?
+
+Unfortunately, from the perspective of a small open-source firmware vendor, it
+isn't easy to work with AMD. Despite our experience with AMD SoCs since 2016,
+we could not yet deliver Dasharo for a modern (Zen core-based) platform. We're
+trying hard, but Intel has a better ecosystem for open-source firmware
+development.
+
+The reason for that state may be because AMD is in a rush, and they are
+understaffed in all areas compared to their success. We've been doing AMD
+open-source firmware development for 6+ years, including our yearly reports of
+open-source firmware status at FOSDEM, but the level of support for small
+volume firmware development companies is not yet at the level of competition.
+
+AGESA distribution was a problem in the past, but we solved that, and Dasharo
+for AMD is possible. Because Dynamic Root of Trust can work without blob, we
+favor AMD, but we can't do anything without a partner who can sponsor the
+development effort. We are on the market of open-source firmware vendors, not
+hardware vendors.
+
+## Can you port Dasharo to my mainboard?
+
+There are other versions of the same questions:
+
+* Dasharo supports mainboard X and I have mainboard Y (or X'). Can you teach me
+  how to port Dasharo to my mainboard?
+* Can you help me port Dasahro to my mainboard?
+
+**TL;DR: No, we can't.**
```

Comment:
In general I agree, but if we have board X' which is a close variant for board X, shouldn't we have some ways to include that to Dasharo family via some community testing?

Reply:
This is very good idea I will add that.